packages/ui/src/components/illustration/illustration.tsx

```diff
@@ -31,6 +31,11 @@ const Illustration: FC<IllustrationProps> = ({
       ? IllustrationsNameMap[`${name}-light` as keyof typeof IllustrationsNameMap]
       : IllustrationsNameMap[name]
 
+  if (!Component) {
+    console.warn(`Icon "${name}" not found in IllustrationsNameMap.`)
+    return null
+  }
+
```

Comment:
Shouldn't this be caught by `typescript` itself? If there's no component mapped against a `name`, it should allow `Illustration` component to be rendered with that `name` itself.

Reply:
Not a blocker though, something that you can discuss separately with @praneshg239 once he's back.